pkgs/development/python-modules/manticore/default.nix

```diff
@@ -0,0 +1,66 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, capstone
+, crytic-compile
+, ply
+, prettytable
+, psutil
+, pyelftools
+, pyevmasm
+, pysha3
+, pyyaml
+, rlp
+, setuptools
+, unicorn
+, wasm
+, wrapt
+, pkgs
+}:
+
+let z3 = pkgs.z3;
+
+in buildPythonPackage rec {
+  pname = "manticore";
+  version = "0.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0f4b6yyg6ssmp4hq5njgxvq25iqyjfsps5k3yyjd6nf2y51lxd78";
+  };
+
+  propagatedBuildInputs = [
+    pyyaml
+    wrapt
+    pysha3
+    prettytable
+    pyevmasm
+    rlp
+    ply
+    crytic-compile
+    wasm
+    psutil
+    setuptools
+    capstone
+    pyelftools
+    unicorn
+  ];
+
+  postPatch = ''
+    sed -ie s/pyevmasm==0.2.0/pyevmasm/ setup.py
+    # Python API is not used in the code, only z3 from PATH
+    sed -ie s/z3-solver// setup.py
+    '';
+
+  makeWrapperArgs = [ ''--prefix PATH : "${z3}/bin"'' ];
+
+  preCheck = ''export PATH=${z3}/bin:$PATH'';
+
+  meta = with lib; {
+    description = "Symbolic execution tool for analysis of smart contracts and binaries";
+    homepage = "https://github.com/trailofbits/manticore";
+    license = licenses.agpl3;
+    platforms = platforms.linux;
+    maintainers = [ maintainers.arturcygan ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ arturcygan ];
```

Reply:
I there a style guide regarding this? I might have missed that

Reply:
No style guide that I know of. Either one would honestly be fine. This just produces slightly smaller diffs in future if add a maintainer, but honestly that's like a 1% use case.